src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -950,21 +950,27 @@ namespace winrt::TerminalApp::implementation
     void CommandPalette::SetActionMap(const Microsoft::Terminal::Settings::Model::IActionMapView& actionMap)
     {
         _actionMap = actionMap;
+        _setCommands();
     }
 
-    void CommandPalette::SetCommands(const Collections::IVector<Command>& actions)
+    void CommandPalette::_setCommands()
     {
-        _allCommands.Clear();
-        for (const auto& action : actions)
+        if (_actionMap)
         {
-            auto actionPaletteItem{ winrt::make<winrt::TerminalApp::implementation::ActionPaletteItem>(action) };
-            auto filteredCommand{ winrt::make<FilteredCommand>(actionPaletteItem) };
-            _allCommands.Append(filteredCommand);
-        }
+            _allCommands.Clear();
```

Comment:
this has a behavior difference - if you call `SetCommands(nullptr)` it clears `allCommands`. Now if you call `SetActionMap(nullptr)` it does _not_ clear `allCommands`

Is this important?

Reply:
I don't think this was ever being called with `nullptr` but I've fixed it to have the same behaviour!